moped-editor/src/views/projects/projectView/ProjectNotes/NoteInput.js

```diff
@@ -273,11 +250,17 @@ const NoteInput = ({
         <Grid item>
           <Box pb={2} display="flex" style={{ justifyContent: "flex-end" }}>
             {isEditingNote && (
-              <div className={classes.cancelButton}>
+              <Box
+                sx={{
+                  margin: (theme) => theme.spacing(1),
+                  position: "relative",
+                  color: (theme) => theme.palette.secondary.dark,
```

Comment:
@mateoclarke could you remove this color while you are in here? I noticed we are getting this same color in staging and this branch which isn't (and shouldn't) apply to this button.

<img width="266" height="102" alt="Image" src="https://github.com/user-attachments/assets/6e3a3fb9-5104-46e8-a7d6-b0685f24da6a" />